core/src/test/java/com/linecorp/armeria/server/Http1ServerDelayedCloseConnectionTest.java

```diff
@@ -82,19 +78,14 @@ void shouldDelayDisconnectByServerSideIfClientDoesNotHandleConnectionClose() thr
             final int readResult = in.read();
             final long readDurationMillis = Duration.ofNanos(System.nanoTime() - readStartTimestamp).toMillis();
 
+            // -1 means that the server closed the connection
+            // after defaultHttp1ConnectionCloseDelayMillis elapsed.
             assertThat(readResult).isEqualTo(-1);
-
             final long defaultHttp1ConnectionCloseDelayMillis = Flags.defaultHttp1ConnectionCloseDelayMillis();
             assertThat(readDurationMillis).isBetween(
                     defaultHttp1ConnectionCloseDelayMillis - 2000,
                     defaultHttp1ConnectionCloseDelayMillis + 2000
             );
-
-            socket.close();
-            try (Socket reuseSock = new Socket()) {
-                assertThatCode(() -> reuseSock.bind(new InetSocketAddress((InetAddress) null, socketPort)))
```

Comment:
We've checked that the server closed the connection after 3 seconds above.
So I think this check isn't necessary which causes a flaky.